src/sc/sidechain.cpp

```diff
@@ -117,18 +117,18 @@ bool Sidechain::checkCertSemanticValidity(const CScCertificate& cert, CValidatio
         return state.DoS(100, error("version too low"), REJECT_INVALID, "bad-cert-version-too-low");
     }
 
-    if (!MoneyRange(cert.totalAmount) || !MoneyRange(cert.GetValueOut()))
+    if (!MoneyRange(cert.totalAmount) || !MoneyRange(cert.GetValueOfBackwardTransfers()))
```

Comment:
Not strictly related to this feature, but it may sufficient to check totalAmount only and skip vouts or the subset of vouts related to backward transfers.
The reason is that CheckCertificate already check range for whole outputs.

Reply:
Ok